pkgs/misc/vim-plugins/generated.nix

```diff
@@ -764,6 +764,17 @@ let
     };
   };
 
+  delimitMate = buildVimPluginFrom2Nix {
```

Comment:
You also need to add this package to vim-plugin-names in the same directory.

Reply:
thanks, got it done